tests/test.py

```diff
@@ -102,6 +102,48 @@ def test_i2c_set_bus_voltage_or_init_bus_are_equivalent(self):
         except Exception as e:
             self.fail(f"I2C read raised an exception {e}")
 
+    def test_i2c_set_pull_up_resistor(self):
+        if self.use_simulator:
```

Comment:
Both tests run ok for me. However, it was strange not to see the Supernova blinking. So, I modified locally the `basic_i2c_example.py` adding this piece of code:
```
    print("SETTING PULL UP RESISTOR")
    time.sleep(3)
    success, result = i2c.set_pull_up_resistors(13232)
    if success:
        print(f"Resistor set in: {result} Ohm")
    else:
        print(f"Operation failed with error: {result}")
    time.sleep(3)
```
And it worked ok with the blinking and everything ;)